packages/android_intent/android/src/main/java/io/flutter/plugins/androidintent/AndroidIntentPlugin.java

```diff
@@ -51,6 +51,29 @@ private String convertAction(String action) {
     }
   }
 
+  private int convertFlag(String flag) {
```

Comment:
This PR converts the flags from String to integer using this method. 
A better method is to let the Dart side know about the flag names & values and simply push the flag through. You can create a new Flags.dart (or similar) and fill in the constants from https://developer.android.com/reference/android/content/Intent (All the constants from FLAG_).

Please follow the hex numbering (0x00400000) and keep the order as it is listed on the documentation page.